vcd_cli/gateway.py

```diff
@@ -48,3 +54,131 @@ def list_gateways(ctx):
         stdout(result, ctx, show_id=False)
     except Exception as e:
         stderr(e, ctx)
+
+
+@gateway.command(short_help='show edge gateway details')
+@click.pass_context
+@click.argument('name',
+                metavar='<name>',
+                required=True)
+def info(ctx, name):
+    restore_session(ctx, vdc_required=True)
+    client = ctx.obj['client']
+    vdc_href = ctx.obj['profiles'].get('vdc_href')
+    vdc = VDC(client, href=vdc_href)
+    edge_gateways = vdc.list_edge_gateways()
+    for gw in edge_gateways:
+        if name == gw.get('name'):
+            resource = client.get_resource(gw.get('href'))
+            result = edge_gateway_to_dict(resource)
+            stdout(result, ctx)
+            return
+    raise Exception('not found')
+
+
+def edge_gateway_to_dict(gw):
+    result = {}
+    result['name'] = gw.get('name')
+    result['href'] = gw.get('href')
+    return result
+
+
+@gateway.command(short_help='show edge gateway nat rules')
+@click.pass_context
+@click.argument('name',
+                metavar='<name>',
+                required=True)
+def natrules(ctx, name):
+    restore_session(ctx, vdc_required=True)
+    client = ctx.obj['client']
+    vdc_href = ctx.obj['profiles'].get('vdc_href')
+    vdc = VDC(client, href=vdc_href)
+    edge_gateways = vdc.list_edge_gateways()
+    for gw in edge_gateways:
+        if name == gw.get('name'):
+            resource = client.get_resource(gw.get('href'))
+            result = edge_gateway_nat_rules_to_dict(resource)
+            stdout(result, ctx)
+            return
```

Comment:
Move this logic to a new method in "vdc.get_nat_rules(edge_gateway_name)". It should ideally go to edge_gateway but as there is no existing class, you can move it to vdc.py.